src/_includes/docs/swift-package-manager/how-to-enable-disable.md

```diff
@@ -0,0 +1,77 @@
+## How to turn on Swift Package Manager
+
+Flutter's Swift Package Manager support is turned off by default.
+To turn it on:
+
+1. Switch to Flutter's `main` channel:
+
+   ```sh
+   flutter channel main
+   ```
+
+2. Make sure you have the latest Flutter SDK:
+
+   ```sh
+   flutter upgrade
+   ```
+
+3. Turn on the Swift Package Manager feature:
+
+   ```sh
+   flutter config --enable-swift-package-manager
+   ```
+
+Running an app using the Flutter CLI migrates it to add Swift Package Manager
+integration.
+This makes your project download the Swift packages that
+your Flutter plugins depend on.
+
+Flutter falls back to CocoaPods for dependencies that do not support Swift
+Package Manager yet.
+
+## How to turn off Swift Package Manager
+
+:::secondary Plugin authors
+Plugin authors need to turn on and off Flutter's Swift Package Manager
+support for testing.
+Otherwise, you likely won't need to turn this off.
+
+If you find a bug in Flutter's Swift Package Manager support,
+please [open an issue][].
+:::
+
+Disabling Swift Package Manager causes Flutter to use CocoaPods for all
+dependencies.
+However, Swift Package Manager remains integrated with your project.
+To remove this integration,
```

Comment:
maybe: "To remove Swift Package Manager completely from your app project,"

disambiguating "this"

Reply:
Updated!